workflow/rules/quant.smk

```diff
@@ -1,28 +1,141 @@
-rule kallisto_index:
+rule kallisto_cds_index:
```

Comment:
Since some time, there are Snakemake wrappers for kallisto quant and kallisto index: https://snakemake-wrappers.readthedocs.io/en/stable/wrappers/kallisto.html

Please change all these rules here to use those wrappers instead of manually defined shell commands and conda envs.

Reply:
Noted. Changed